events/src/eventsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
     "log-level": "INFO",
-    "port": 24009,
+    "port": 55555,
     "disable-events-log": false
 }
```

Comment:
This file will not be replaced on upgrade. Documentation is required to update this Port.

Reply:
Ack. I'll send out a patch for handling the option in glusterdocs.